flight/Modules/UAVOMSPBridge/UAVOMSPBridge.c

```diff
@@ -218,7 +219,7 @@ static void _msp_send_attitude(struct msp_bridge *m)
 
 	data.att.x = attActual.Roll * 10;
 	data.att.y = attActual.Pitch * -10;
-	data.att.h = 0 * 10;
+	data.att.h = attActual.Yaw;
```

Comment:
Something isn't symmetric here. There is a ```*10``` multiplier for Roll and Pitch, but not for yaw. Is this normal?

No matter if it is or isn't, can you please add comments that explain clearly why there are rescaling multipliers?